spec/models/vote_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+RSpec.describe Vote, type: :model do
+  it { should belong_to(:user) }
+  it { should belong_to(:review) }
+  it { should validate_presence_of(:user) }
+  it { should validate_presence_of(:review) }
+  let(:user) { User.create(email: "goofy@gmail.com", password: 'password') }
+  let(:vendor) { Vendor.create(vendor_name: "Yum") }
+  let(:review) { Review.create(
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.